sympy/parsing/tests/test_autolev.py

```diff
@@ -14,22 +14,28 @@
     os.path.dirname(os.path.abspath(os.path.realpath(__file__))))
 
 
-def _test_examples(in_filename, out_filename, test_name=""):
+def _test_examples(in_filename, out_filename, test_name="", include_numpy=False):
 
     in_file_path = os.path.join(FILE_DIR, 'autolev', 'test-examples',
                                 in_filename)
     correct_file_path = os.path.join(FILE_DIR, 'autolev', 'test-examples',
                                      out_filename)
     with open(in_file_path) as f:
-        generated_code = parse_autolev(f, include_numeric=True)
+        generated_code = parse_autolev(f, include_numeric=True, include_numpy=include_numpy)
 
     with open(correct_file_path) as f:
         for idx, line1 in enumerate(f):
             if line1.startswith("#"):
                 break
             try:
                 line2 = generated_code.split('\n')[idx]
-                assert line1.rstrip() == line2.rstrip()
+                if not include_numpy:
+                    assert "numpy" not in line2
+                else:
+                    math_import = "math as m"
```

Comment:
I wouldn't recommend importing as m. m is very likely to be variable name. Just do `import math`. That's fine.

Reply:
It is done